lib/truffle/java/java.rb

```diff
@@ -122,6 +127,10 @@ def inspect
       super.to_s
     end
 
+    def const_get(name, inherit=true)
+      super.const_get(name, inherit) rescue method_missing(name)
```

Comment:
super.const_get does not work, that's Java syntax.
This whole method should not needed I think.